torch/csrc/jit/api/module.h

```diff
@@ -237,7 +238,12 @@ struct TORCH_API Module : public Object {
   // function creates a new `ClassType` and returns a new instance that has the
   // same data as the current instance but with the new type, shared ClassType
   // will be preserved as well
-  Module clone(bool inplace = false) const;
+  Module clone(
+      bool inplace = false,
+      const std::unordered_set<std::string>& ignored_methods =
+          std::unordered_set<std::string>(),
+      const std::unordered_set<std::string>& ignored_attributes =
+          std::unordered_set<std::string>()) const;
```

Comment:
It might be better to create a new method instead of changing API and implementation of `clone`. In the future, when a proper solution lands we can more easily revert this work-around.